backend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lmrdashboard-backend",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
The version upgrade should only happen after it has been merged into the main branch. So we can see if the changes work on staging. If that is the case, the release/version upgrade can be made so it's deployed to prod.

Reply:
rolled back version numbers